File: JAVA/2186. Minimum Number of Steps to Make Two Strings Anagram II.java
class Solution {
    public int minSteps(String s, String t) {
        int[] a=new int[26];
        int[] b=new int[26];
        for(int i=0;i<s.length();i++)
        a[s.charAt(i)-'a']++;
        for(int i=0;i<t.length();i++)
        b[t.charAt(i)-'a']++;
        int c=0;
        for(int i=0;i<26;i++)
        c+=Math.abs(b[i]-a[i]);
        return c;
    }
}

File: C++/2186. Minimum Number of Steps to Make Two Strings Anagram II.cpp
class Solution {
public:
    int minSteps(string s, string t) {
        vector<int>a(26,0),b(26,0);
        for(char c:s)
        a[c-'a']++;
        for(char c:t)
        b[c-'a']++;
        int k=0;
        for(int i=0;i<26;i++)
        k+=abs(a[i]-b[i]);
        return k;
    }
};